backend/app/models/__init__.py

```
@@ -0,0 +1,5 @@
+from sqlmodel import SQLModel
+
+from .request import RequestLog
+
+from .validator import ValidatorLog
```

Ruff (0.14.10)

No newline at end of file

Add trailing newline

(W292)